src/View/Includes/_footer.html.twig

```diff
@@ -0,0 +1,35 @@
+<div class="footer-dark">
+    <footer>
+        <div class="container">
```

Comment:
why this extra div in the footer? couldn't those classes be directly applied to the footer tag?